examples/sed_tophat.yaml

```diff
@@ -0,0 +1,108 @@
+modules:
+  - galsim_extra
+
+stamp:
+  type: Basic
+
+psf:
+  type: ChromaticAtmosphere
+  base_profile:
+      type: Gaussian
+      fwhm: 0.9
+  base_wavelength: 500
+  zenith_angle: 0 deg  # set to 0 to disable DCR
+  parallactic_angle: 0 deg  # set to 0 to disable DCR
+
+gal:
+  type: Exponential
+  half_light_radius: 0.5
+  signal_to_noise: 1e6
+
+  sed:
+    type: Tophat
+    bins: [[  1000.        ,   1246.        ],
+           [  1246.        ,   1552.        ],
+           [  1552.        ,   1933.        ],
+           [  1933.        ,   2407.        ],
+           [  2407.        ,   2998.        ],
+           [  2998.        ,   3184.        ],
+           [  3184.        ,   3381.        ],
+           [  3381.        ,   3590.        ],
+           [  3590.        ,   3812.        ],
+           [  3812.        ,   4048.        ],
+           [  4048.        ,   4299.        ],
+           [  4299.        ,   4565.        ],
+           [  4565.        ,   4848.        ],
+           [  4848.        ,   5148.        ],
+           [  5148.        ,   5467.        ],
+           [  5467.        ,   5806.        ],
+           [  5806.        ,   6166.        ],
+           [  6166.        ,   6548.        ],
+           [  6548.        ,   6954.        ],
+           [  6954.        ,   7385.        ],
+           [  7385.        ,   7843.        ],
+           [  7843.        ,   8329.        ],
+           [  8329.        ,   8846.        ],
+           [  8846.        ,   9395.        ],
+           [  9395.        ,   9978.        ],
+           [  9978.        ,  11467.        ],
+           [ 11467.        ,  13177.        ],
+           [ 13177.        ,  15143.        ],
+           [ 15143.        ,  17402.        ],
+           [ 17402.        ,  19998.        ]]
+    values: [   1697.29638672 ,
```

Comment:
These lists inputs are going to be impossible to work with, right?

I think we'll need something that can take a column from a catalog.